techniques/failures/F40.html

```diff
@@ -154,27 +146,27 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
 
 
 
-<section id="related"><h2>Related Techniques</h2><ul>
+<section id="related"><h2>関連テクニック</h2><ul>
       <li><a href="../server-side-script/SVR1">SVR1: Implementing automatic redirects on the server side instead of on the
 	client side</a></li>
       <li><a href="../html/H76">H76: Using meta refresh to create an instant client-side redirect</a></li>
    </ul></section>
-<section id="tests"><h2>Tests</h2>
-      <section class="procedure" id="procedure"><h3>Procedure</h3>
-        <p>For a page that uses <code class="language-html">meta http-equiv="refresh"</code>:</p> 
+<section id="tests"><h2>検証</h2>
+      <section class="procedure" id="procedure"><h3>手順</h3>
+        <p><code class="language-html">meta http-equiv="refresh"</code> を使用しているページについて:</p> 
         <ol>
-          <li>Check that the numerical value for seconds until refresh in the <att>content</att> attribute is present.</li>
-          <li>Check that the numerical value for seconds until refresh in the <att>content</att> attribute is less than one or greater than 72,000.</li>
-          <li>Check if the page qualifies for Real-time or Essential Exceptions in <a href="/TR/WCAG/#timing-adjustable">Success Criterion 2.2.1 Timing Adjustable</a>.</li>
-          <li>Check if the user is provided an opportunity to turn off, extend, or adjust the timing of the page refresh.</li>
-            <li>Check if the page does not redirect after the duration specified in the <att>content</att> attribute.</li>
+          <li><att>content</att> 属性の中に更新までの秒数の数値が存在することを確認する。</li>
```

Comment:
check that ... の訳を他のテクニックと合わせました。
```suggestion
          <li><att>content</att> 属性の中に更新までの秒数の数値が存在することをチェックする。</li>
```